030-Concepts/080-pricing.mdx

```diff
@@ -7,26 +7,195 @@ slug: concepts/pricing
+- **Security:** Single Sign-On (SSO) for secure and efficient access management.
+- **Support:** Dedicated support to receive support tailored to your projects specific needs and challenges.
+
+
+### Change plans
+
+Visit the [pricing page](https://xata.io/pricing) to upgrade from the Free to Pro plan. For the Enterprise plan, please contact us.
+
+
+## FAQ
+
+<Expand title="Usage limits and activity ">
+##### Is there a maximum number of reads and writes per month?
+No, you can serve as many reads and writes as you want within the rate limit, which is defined per second (resets every second).
+
+##### What happens if I exceed the rate limit?
+If you exceed the rate limit on Xata, you will receive a 429 HTTP status as a notification of the overload. Xata’s pricing structure is specifically tailored to cater to diverse user needs, from individual endeavors to large-scale enterprise applications. Detailed information about pricing, billing, and usage limits is available in the documentation on this page. Additionally, our [pricing page](https://xata.io/pricing) offers a thorough breakdown of our pricing model.
+
+##### Can I increase my rate limit?
+Yes, you can increase the limit by upgrading to the Pro plan, if you are on the Free plan. You can also contact us if you would like to discuss pricing options.
+
+##### How are read and write requests measured?
+
+##### Do the reads and writes per second differ depending on the plan chosen?
+
+##### How is the data size measured?
+We measure and determine data size by database storage, search engine storage, file attachment storage.
+
+##### What happens when the rate limit or concurrency limits are reached?
+When the rate limit or concurrency limits are reached, we will throttle your requests and  answer with a 429 status code. Your app will be in a degraded mode, but only the requests over the limit are affected. The rate limit is reset every second, and the concurrency limit is measured at any given time.
+
+</Expand>
+
+<Expand title="Changing plans">
+##### What happens on upgrade and downgrade?
+If you decide to upgrade or downgrade your subscription plan in the middle of a billing cycle, we calculate the charges or credits based on the number of days remaining in the month. This approach ensures that you are billed accurately, reflecting the services you use during the billing cycle.
+
+##### Will my database be suspended after a period of inactivity?'
+Nope. Even if it’s not accessed, we don’t put your database to sleep, even if your'e on the free plan. We want projects of all kinds to be able to depend on Xata even if they are not paying.'
+
+</Expand>
+
+<Expand title="Billing and special pricing">
+##### When will I be billed?
+For our paid plans, you will receive an invioce on the first day of each month. This will include usage from the previous month and any outstanding balances.
```

Comment:
I think this is not entirely correct, because we charge the $20 when you upgrade, and then we charge always on the same date each month (not on the first). I think @philkra knows the details here better. But I'm not sure we need to document this, as it's subject to change.